react/features/virtual-background/components/VirtualBackgroundDialog.js

```diff
@@ -119,6 +126,17 @@ function VirtualBackground({ _jitsiTrack, _selectedThumbnail, dispatch, t }: Pro
         });
     };
 
+    const shareDesktop = async selection => {
+        const url = await createLocalTrack('desktop', '');
```

Comment:
The general practice is to use one of the functions for track creation from /features/base/track. What happens here when the track creation fails ?

Reply:
Added check for url.

Reply:
```
react_devtools_backend.js:2560 2021-05-26T14:14:37.265Z [JitsiMeetJS.js] <Object.getGlobalOnErrorHandler>:  UnhandledError: Could not create desktop local track! Script: null Line: null Column: null StackTrace:  Error: Could not create desktop local track!
    at shareDesktop (https://localhost:8080/libs/app.bundle.min.js?v=5031:292097:11)
overrideMethod @ react_devtools_backend.js:2560
o @ Logger.js:154
getGlobalOnErrorHandler @ JitsiMeetJS.js:545
window.onunhandledrejection @ middleware.js:127
VirtualBackgroundDialog.js:135 Uncaught (in promise) Error: Could not create desktop local track!
    at shareDesktop (VirtualBackgroundDialog.js:135)
```
The error needs to be handled and a UI notification that desktop share has failed needs to be shown so that the user is aware something went wrong, otherwise the user would keep clicking on desktop share and it wouldn't get applied.

Reply:
Added error notification + logger + new translation field